src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = ",";
+    private static final String SEPARATOR = System.lineSeparator();
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String contents = readFromFile(fromFileName);
+        String report = getReport(contents);
+        writeReport(report, toFileName);
+    }
+
+    public static String readFromFile(String file) {
+        StringBuilder builder = new StringBuilder();
```

Comment:
For the future, do not use such naming(builder, value, stringBuilder, str), how to guess what it stores?